paper/paper.md

```diff
@@ -0,0 +1,109 @@
+by now also supports other neuroimaging modalities such as MEG [@Niso2018] and
+EEG [@Pernet2019].
+
+BIDS prescribes how complex experiment data should be structured and which
+metadata should be encoded next to the raw data. This set of guidelines
+allows users to reap several benefits, such as:
+
+1. **sharing data within a lab and between labs:** Through reading the BIDS
+   specification, users are empowered to understand any BIDS dataset
+   without requiring specialist knowledge.
+1. **data validation:** Dedicated software can check whether the rules of
+   BIDS are being followed in a dataset, and alert users to missing or broken
+   data (see
+   [bids-validator](https://github.com/bids-standard/bids-validator)).
+1. **automatic data handling and analysis pipelines:** With a perfect
+   knowledge about which data to expect and where to search for it, many
+   common workflows in data handling and analysis can be simplified or
+   automated through specialized software.
+
+To make use of these benefits however, datasets first must be converted to
+BIDS format. Furthermore, already existing data analysis software must be
+extended to be aware of a BIDS format and how to properly use it.
+
+This is where ``MNE-BIDS`` steps in: Harnessing BIDS and the data analysis
+software MNE-Python [@Agramfort2013], it is the goal of ``MNE-BIDS`` to
+automatically organize raw MEG and EEG data into BIDS format and to facilitate
+their analysis.
+
+Starting with, for example, a single directory full of datafiles with arbitrary
+naming, ``MNE-BIDS`` can be used to extract present meta data, reorganize the
+files into BIDS format, and write additional meta data. Moreover,
+``MNE-BIDS`` supports conversion from raw data formats that are not BIDS
+compatible into permissible formats. As a result, users can easily convert
+their datasets to BIDS in a matter of minutes, rather than after hours of
+manual labour.
+
+Next to this core functionality, ``MNE-BIDS`` is continuously being extended
+to facilitate the analysis of BIDS formatted data. To name some features, it is
+possible to read a raw BIDS dataset and obtain a Python object, ready for
+analyis with MNE-Python, or to save a T1 weighted anatomical MRI image
+alongside the MEG or EEG data and apply an automatic defacing algorithm for
+anonymization purposes
+
+Users can easily install ``MNE-BIDS`` on all platforms via `pip` and `conda`
+and its functionality is continuously tested on Windows and Linux.
+Next to three core dependencies for scientific computation (`numpy`, `scipy`),
+and handling of MEG and EEG data (`mne`), ``MNE-BIDS`` has only minimal
+dependencies, all of which are optional. The API of the package is stable and
+extensively documented and explained in examples (https://mne.tools/mne-bids/).
+
+As of writing, ``MNE-BIDS`` has received code contributions from 15
+contributors and its' user base is steadily growing. Code development is
```

Comment:
```suggestion
contributors and its user base is steadily growing. Code development is
```

Reply:
but it's the user base of the package (it) ... so its' --> this should be correct.

Reply:
not sure I follow. The possessive "its" does not have an apostrophe unlike for example "Mainak's" or "Stefan's"

Reply:
haha, okay I checked this thoroughly now ... what I suggested does not exist. Thanks :-D